File: 2 Revalida en C++/3 Decisiones/7.cpp
///para saber si tres lados formanun triangulo
#include <bits/stdc++.h>
using namespace std;
int main()
{
    int a,b,c,s;
    cout<<"Ingrese a:";
    cin>>a;
    cout<<"Ingrese b:";
    cin>>b;
    cout<<"Ingrese c:";
    cin>>c;
    int max;
    int min;
    if (a>b)
       max=a;
    else
        max=b;
    if (max<c)
        max=c;
    s=(a+b+c)/2;
    if (s>max)
        cout<<"forman triangulo";
    else
        cout<<"no forman triangulo";
    return 0;
}

File: 1 Revalida en python/3 Decisiones/7.py
#para saber si tres lados formanun triangulo
a=int(input("ingrese un numero"))
b=int(input("ingrese un numero"))
c=int(input("ingrese un numero"))
if a>b:
	max=a
else:
	max=b
if max<c:
	max=c
s=(a+b+c)/2
if s>max:
	print"forman triangulo"
else:
	print"no forman triangulo"